meters/rs485/sdm54.go

```diff
@@ -0,0 +1,90 @@
+package rs485
+
+import . "github.com/volkszaehler/mbmd/meters"
+
+func init() {
+	Register("SDM54", NewSDM54Producer)
+}
+
+type SDM54Producer struct {
+	Opcodes
+}
+
+func NewSDM54Producer() Producer {
+	/**
+	 * Opcodes as defined by Eastron SDM54.
+	 * See https://www.eastrongroup.com/eastrongroup/2024/08/21/eastronsdm54seriesusermanualv1.2.pdf
+	 * This is to a large extent a superset of all SDM devices, however there are
+	 * subtle differences (see 220, 230). Some opcodes might not work on some devices.
+	 */
+	ops := Opcodes{
```

Comment:
Please `gofmt` the code (or use VScode with Go plugin)

Reply:
Thanks for your comment, I have done that. As this is my first time adapting go code I would need some support: can the for

Reply:
OK, I figured it out. Is there anything else?